libs/nss/Makefile

```diff
@@ -0,0 +1,115 @@
+#
+# Copyright (C) 2019 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nss
+PKG_VERSION:=3.43
+_PKG_VERSION:=3_43
```

Comment:
This is ugly, remove and hardset it instead

Reply:
I think I saw somewhere a replace function so I only have to change in one place, I'll look into this